catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/suggestion/UtmUpsCoordinateProcessorTest.java

```diff
@@ -301,16 +278,27 @@ private void assertSuggestionDoesNotExist(String query) {
     assertThat(list, is(empty()));
   }
 
-  private void assertSuggestion(String query, String expectedName, List<LatLon> expectedGeo) {
+  private void assertSuggestion(String query, String expectedName, double... expectedLatLons) {
     List<Suggestion> list = new LinkedList<>();
     processor.enhanceResults(list, query);
-    assertThat(list, is(not(empty())));
     assertThat(list, hasSize(1));
 
     LiteralSuggestion literalSuggestion = (LiteralSuggestion) list.get(0);
-    assertThat(literalSuggestion.getId(), is("LITERAL-UTM-UPS"));
-    assertThat(literalSuggestion.getName(), is(expectedName));
-    assertThat(literalSuggestion.hasGeo(), is(true));
-    assertThat(literalSuggestion.getGeo(), is(equalTo(expectedGeo)));
+    assertSuggestion(literalSuggestion, expectedName, expectedLatLons);
+  }
+
+  private void assertSuggestion(
+      LiteralSuggestion actualSuggestion, String expectedName, double... expectedLatLons) {
```

Comment:
✏️ We could use some nice, specific failure messages for all the following `assertThat` statements. So we know why something failed after we've all forgotten how this works.